app/src/main/java/org/projectbuendia/client/ui/chart/PatientChartActivity.java

```diff
@@ -355,7 +321,7 @@ private String getFormattedPcrString(double pcrValue) {
                     .color(0x00000000)
                     .sizeDp(36));
             if ((pcrLObservation == null || pcrLObservation.valueName == null)
-                && (pcrNpObservation == null || pcrNpObservation == null)) {
+                && (pcrNpObservation == null || pcrNpObservation.valueName == null)) {
```

Comment:
I'm not sure what was going on previously, but it was definitely a typo that triggered an inspection in my IDE, which is why I looked into it. The logic for pcrNpObservation is supposed to mirror pcrLObservation here, but .valueName had been left off erroneously.